### ISSUE TITLE
Attempt at Strikeout in documentation struck out and left ~~s lying around

### ISSUE THREAD

**Issue by jsoref** [argoproj/argo-cd#14139]
Checklist:

* [ ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [ ] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

There are `~~`s around text which apparently wanted to be stricken.

**To Reproduce**

1. Visit https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.4-2.5/#argocd-cm-plugins-cmps-are-deprecated

**Expected behavior**

Stricken text

**Screenshots**

<img width="976" alt="image" src="https://github.com/argoproj/argo-cd/assets/2119212/9b62e1d4-3940-4f77-b005-7165b89627e3">

**Version**

```shell
Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by rkriplani76** [comment]
I can work on this provided, this issue is still relevant as it should have been depreciated.

**Comment by crenshaw-dev** [comment]
@rkriplani76 yep, still relevant. :-)

**Comment by jsoref** [comment]
Fixed by #14823
